### ISSUE TITLE
[Feature Request] Bock beer

### ISSUE THREAD

**Issue by Deshiba** [GrowthcraftCE/Growthcraft-1.12#124]
I'd love the ability to create bock beers by adding Hops to Lager in a brew kettle with increasing strengths of intoxication.
- Lager + Hop = Bock
- Maibock + Hop = Doppelbock
- Doppelbock + Ice = Eisbock

This mechanic would be roughly in line with how these beers are created irl.https://en.wikipedia.org/wiki/Bock

**Comment by Deshiba** [comment]
Brewing is always interesting 😄However the trick is not just adding ice itself, to quote the article it's beer "...that is made by partially freezing a doppelbock and removing the water ice to concentrate the flavour and alcohol content..."

I thought using an ice block would be easier as a crafting recipe.  A more involved brewing method of placing ice next to or underneath the brewing kettle would be preferred.

**Comment by Syndaryl** [comment]
Underneath the brew kettle, replacing the fire block, makes the most consistent sense.
Secondary output would probably be a snowball.

**Comment by Deshiba** [comment]
I agree with Syndaryl on this one. The problem with making it cold region dependent is 2-fold:
1. With certain maps and large biomes you wouldn't necessarily have nearby cold biomes.
2. If you brew Maibock with Hops to get Doppelbock in a cold biome it would automatically freeze to get Eisbock.

There's probably ways around number 2, but the biome dependency seems like more effort for less benefit to me.
